src/prostredi/editor/clion/cmake.md

```diff
@@ -0,0 +1,92 @@
+## Co je CMake?
```

Comment:
Ve skriptech už je sekce o CMake (https://mrlvsb.github.io/upr-skripta/c/automatizace_prekladu.html), bylo by fajn to sjednotit. Asi bych vzal to, cos tady napsal, zkombinoval to s tím, co už je v sekci "automatizace překladu", a nechal to v té sekci automatizace překladu.

Reply:
Tady by bylo nejlepší, pokud by jsi to upravil ty. Nevím co přesně tam chcete/nechcete. Nechal jsem ti tam ten můj cmake.md tak ho pak můžeš smazat